packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -30,6 +23,14 @@ import {
 import type { TranslationKey } from 'src/components/suite/Translation/components/BaseTranslation';
 import { createAction } from '@reduxjs/toolkit';
 
+import { notificationsActions } from '@suite-common/toast-notifications';
+import { getCustomBackends } from '@suite-common/wallet-utils';
+import { desktopApi, HandshakeElectron } from '@trezor/suite-desktop-api';
+import { analytics, EventType } from '@trezor/suite-analytics';
+import TrezorConnect, { Device, DEVICE } from '@trezor/connect';
```

Comment:
nit: those imports above `suite` imports.

Reply:
This is actually some kind of linting that happened automatically.